src/Microsoft.Diagnostics.Monitoring.WebApi/LoggingExtensions.cs

```diff
@@ -51,6 +52,18 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Warning,
                 formatString: Strings.LogFormatString_DefaultProcessUnexpectedFailure);
 
+        private static readonly Action<ILogger, string, string, Exception> _stoppingTraceEventHit =
+            LoggerMessage.Define<string, string>(
+                eventId: new EventId(8, "StoppingTraceEventHit"),
+                logLevel: LogLevel.Information,
```

Comment:
```suggestion
                logLevel: LogLevel.Debug,
```

I don't think we need to log this normally since the action executor should be logging the completion of the action.